client/src/components/EventForm/index.jsx

```diff
@@ -58,36 +86,54 @@ class AddEvent extends Component {
       start_date: moment(
         [this.state.start_date, this.state.start_time].join(' '),
         'D MMMM, YYYY h:mmA'
-      )._d.toJSON(),
+        )._d.toJSON(),
```

Comment:
I don't understand this "_d.toJSON" why it have "_d."?

Reply:
moment() return an object, but all I need it is a string with datetime in an appropriate to JSON format.
![image](https://user-images.githubusercontent.com/18135784/44431232-e41b9580-a5a5-11e8-8e5a-b01b468239be.png)

Reply:
But why that have so weird cut name of date?

Reply:
Please ask these question to the moment.js creators, not to me.